go/pkg/sysdb/metastore/db/dao/collection.go

```diff
@@ -142,6 +143,29 @@ func (s *collectionDb) getCollections(id *string, name *string, tenantID string,
 	return
 }
 
+func (s *collectionDb) GetCollectionSize(id string) (uint64, error) {
```

Comment:
Is there some magical reason why some of these methods take a `*string` versus ones like this that take a `string`? I don't have enough context on why one would make sense over the other but it would be nice to be consistent. I feel like `string` should be the way to go here, but the callsite for this is using an interface which requires a deref, which is not great.

Reply:
Updated the call site to not require a deref. I think the use of `*string` in some places is because it isn't a required param.